ValetTests/ValetTests.m

```diff
@@ -90,15 +91,20 @@ - (void)setUp;
     self.string = @"bar";
     self.secondaryString = @"bar2";
     self.additionalValets = [NSMutableArray new];
+    self.manualKeychainEntries = [NSMutableArray<NSDictionary *> new];
 }
 
 - (void)tearDown;
 {
-    [super tearDown];
-    
     for (VALValet *const additionalValet in self.additionalValets) {
         [additionalValet removeAllObjects];
     }
+
+    for (NSDictionary *keychainEntry in self.manualKeychainEntries) {
```

Comment:
Seems like a reasonable solution! Since this cleanup is needed for only one test may be best to keep the cleanup localized to that test. I'm a big fan of putting the cleanup at the top of the test, so if tearDown is aborted the next run of the test still succeeds.